allennlp/tests/data/tokenizers/pretrained_transformer_tokenizer_test.py

```diff
@@ -0,0 +1,12 @@
+# pylint: disable=no-self-use,invalid-name
+
+from allennlp.common.testing import AllenNlpTestCase
+from allennlp.data.tokenizers import PretrainedTransformerTokenizer
+
+class TestPretrainedTransformerTokenizer(AllenNlpTestCase):
+    def test_splits_into_wordpieces(self):
+        tokenizer = PretrainedTransformerTokenizer('bert-base-cased', do_lowercase=False)
```

Comment:
It would be good to not have to the dependency on the web for the test, but I'm pretty sure it wouldn't work, because we rely on pytorch-transformers' `AutoTokenizer`.

Well, *maybe* it would work if I make sure the filename that I give for the fixture passes their heuristics for which actual tokenizer to use, but (1) I'm not sure that I can, because their heuristics might not allow it, and (2) that would make the test too dependent on the internals of their implementation.